release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -81,6 +81,11 @@ For more information, see xref:../getting_started/openshift-overview.adoc[Gettin
 
 {op-system} now supports `kdump` for the `arm64` architecture in {product-title} 4.11 as a Technology Preview.
 
+[id="ocp-4-11-rhcos-rhel-8-6-packages"]
+==== {op-system} now uses {op-system-base} 8.6
+
+{op-system} now uses {op-system-base-full} 8.6 packages in {product-title} 4.11 and above. This enables you to have the latest fixes, features, and enhancements, as well as the latest hardware support and driver updates.
```

Comment:
I think I like what you have better, but I thought I'd leave a small suggestion for that last sentence. I am not sure if there is a way to make it feel less "listy" though. 
```suggestion
{op-system} now uses {op-system-base-full} 8.6 packages in {product-title} 4.11 and above. This enables you to have the latest fixes, features and enhancements, hardware support, and driver updates. 
```